data/discovery/registry/national-health-service.yaml

```diff
@@ -0,0 +1 @@
+registry: national-health-service
```

Comment:
Discussed this with @rich-vale and we think this should be `nhs-commissioning-board`. And that's in the list!